hms-app/src/components/MoveIdeaModal.tsx

```diff
@@ -12,9 +12,12 @@ import { removeIdeaParticipant } from '../actions/ParticipantActions'
 
 type IProps = {
   idea: Idea
+  reloadIdeaList?: () => void
```

Comment:
can we rename this into "onSuccess"

Reply:
oh and it doesnt need the "?" as we only call MoveIdeaModal only once and when then we pass that for sure